core/src/data/properties.cpp

```diff
@@ -31,14 +31,22 @@ void Properties::setSorted(std::vector<Item>&& _items) {
 const Value& Properties::get(const std::string& key) const {
     const static Value NOT_FOUND(none_type{});
 
-    const auto it = std::lower_bound(props.begin(), props.end(), key,
-                                     [](const auto& item, const auto& key) {
-                                         return keyComparator(item.key, key);
-                                     });
-
-    if (it == props.end() || it->key != key) {
+    const auto it = std::find_if(props.begin(), props.end(),
+                                 [&](const auto& item) {
+                                     return item.key == key;
+                                 });
+    if (it == props.end()) {
         return NOT_FOUND;
     }
+
+    // auto it = std::lower_bound(props.begin(), props.end(), key,
+    //                            [](auto& item, auto& key) {
+    //                                return keyComparator(item.key, key);
+    //                            });
+    // if (it == props.end() || it->key != key) {
+    //     return NOT_FOUND;
+    // }
+
     return it->value;
 }
```

Comment:
@tallytalwar If you mean this one. Can we leave it in as a reminder to test which one is faster on different platforms?